corehq/apps/consumer_user/signals.py

```diff
@@ -0,0 +1,55 @@
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
+
+from corehq.apps.consumer_user.models import (
+    ConsumerUserCaseRelationship,
+    ConsumerUserInvitation,
+)
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.form_processor.signals import sql_case_post_save
+
+from .const import (
+    CONSUMER_INVITATION_ACCEPTED,
+    CONSUMER_INVITATION_CASE_TYPE,
+    CONSUMER_INVITATION_SENT,
+    CONSUMER_INVITATION_STATUS,
+)
+from .tasks import create_new_invitation
+
+
+def send_email_case_changed_receiver(sender, case, **kwargs):
+    if case.type != CONSUMER_INVITATION_CASE_TYPE:
+        return
+    email = case.get_case_property('email')
+    status = case.get_case_property(CONSUMER_INVITATION_STATUS)
+
+    host_indices = [index for index in case.indices if index.relationship == CASE_INDEX_EXTENSION]
+    try:
+        demographic_case_id = host_indices[0].referenced_id
+    except IndexError:
+        return
```

Comment:
Yes, originally I had thought this would write back to the invitation case whenever an error was encountered. I'm going to add that functionality in.